feature/gribi/otg_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -99,18 +108,14 @@ func TestIPv4Entry(t *testing.T) {
 	configureDUT(t, dut)
 
 	ate := ondatra.ATE(t, "ate")
-	ateTop := configureATE(t, ate)
-
-	port2Flow := createFlow("Port 1 to Port 2", ate, ateTop, &atePort2)
-	port3Flow := createFlow("Port 1 to Port 3", ate, ateTop, &atePort3)
-	ecmpFlow := createFlow("Port 1 to Port 2 & 3", ate, ateTop, &atePort2, &atePort3)
+	configureATE(t, ate)
 
 	cases := []struct {
 		desc                 string
 		entries              []fluent.GRIBIEntry
 		downPort             *ondatra.Port
-		wantGoodFlows        []*ondatra.Flow
-		wantBadFlows         []*ondatra.Flow
+		wantGoodFlows        []string
```

Comment:
why convert them?

Reply:
ondatra.Flow uses ondatra.Headers which don't work in OTG/gosnappi configurations. Here the flows are simple names which we could use for stats. With ondatra/OTG we can't start traffic on individual flows